waspc/data/Generator/templates/server/src/routes/operations/_query.js

```diff
@@ -1,28 +1,5 @@
 {{={= =}=}}
-import { 
-  deserialize as superjsonDeserialize,
-  serialize as superjsonSerialize,
-} from 'superjson'
-import { handleRejection } from '../../utils.js'
+import { withOperationsMiddleware } from '../../middleware/operations.js'
 {=& operationImportStmt =}
 
-export default handleRejection(async (req, res) => {
-  {=! TODO: When generating express route for query, generated code would be most human-like if we
```

Comment:
We discussed this offline, and since this comment is captured in this issue, it's no longer necessary here https://github.com/wasp-lang/wasp/issues/445